client/service/src/metrics.rs

```diff
@@ -34,7 +35,7 @@ struct PrometheusMetrics {
 	ready_transactions_number: Gauge<U64>,
 
 	// I/O
-	network_per_sec_bytes: GaugeVec<U64>,
+	network_bytes: GaugeVec<U64>,
```

Comment:
I believe that we can just remove this field altogether. In other words, no longer report the bandwidth usage from `client/service` at all.

It is completely redundant with the metric reported by `sc-network`.